web/yarn.lock

```diff
@@ -3316,7 +3316,7 @@ libsodium-wrappers@0.7.9:
   dependencies:
     libsodium "^0.7.0"
 
-libsodium@^0.7.0:
+libsodium@0.7.9, libsodium@^0.7.0:
```

Comment:
This dependency change is unnecessary?

Reply:
Actually, I haven't made any dependency changes. Maybe It happened while running it locally like yarn install ?

Reply:
Perhaps, could you please revert the changes in this file? It'd be best to reduce the scope of the PR. Thank you!